tests/forcetorque/ForceTorqueTest.cc

```diff
@@ -36,7 +36,15 @@ TEST(ForceTorqueTest, PluginTest)
   double timestamp;
   
   isixaxis->getSixAxisForceTorqueSensorName(0, sensorName);
-  isixaxis->getSixAxisForceTorqueSensorMeasure(0, measure, timestamp);
+  //isixaxis->getSixAxisForceTorqueSensorMeasure(0, measure, timestamp);
+  size_t maxNrOfReadingAttempts = 20;
+  bool readSuccessful = false;
+  for (size_t i=0; (i < maxNrOfReadingAttempts) && !readSuccessful ; i++)
+  {
+      readSuccessful = isixaxis->getSixAxisForceTorqueSensorMeasure(0, measure, timestamp);
+      std::this_thread::sleep_for(std::chrono::milliseconds(100));
+  }
+	ASSERT_TRUE(readSuccessful);
```

Comment:
Please avoid mixing tabs and spaces, as it may result in inconsistent rendring (the width of tabs is not constant).
```suggestion
  ASSERT_TRUE(readSuccessful);
```